api/nodes/serializers.py

```diff
@@ -154,9 +155,28 @@ def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
     def get_unread_comments_count(self, obj):
```

Comment:
I think I made it opt-in, but I don't think it is opt-in any more. I will change it back to opt-in, since getting the number of unread comments is non-trivial.